models/issue_tracker.go

```diff
@@ -0,0 +1,118 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"github.com/VojtechVitek/go-trello"
+	"github.com/google/go-github/github"
+	"log"
+	"strings"
+)
+
+type Configuration struct {
+	ApiKey   string
+	Token    string
+	UserName string
+}
+
+type TrelloIssueProvider struct {
+	Configuration
+	BoardId  string
+	ListName string
+}
+
+type GithubIssueProvider struct {
+	Query string
+}
+
+type Issue struct {
+	title       string
+	description string
+}
+
+func PrintIssue(issue Issue) {
+	fmt.Println("title: ", string(issue.title))
+	fmt.Println("issue: ", string(issue.description))
+	fmt.Println("")
+}
+
+func (t TrelloIssueProvider) FetchData(c chan Issue) {
+	trello, err := trello.NewAuthClient(t.Configuration.ApiKey, &t.Configuration.Token)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	// @trello Boards
+	board, err := trello.Board(t.BoardId)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	// @trello Board Lists
+	lists, err := board.Lists()
+	if err != nil {
+		log.Fatal(err)
```

Comment:
As this is a library, shouldn't we return an error instead of killing the program?